core/config/initializers/00_configuration.rb

```diff
@@ -127,11 +127,6 @@
   end
 
   fieldset 'Search', namespaced: false do
-    field 'Default Search Facet Result Sizes',
-      type: :integer,
-      default: 10,
-      description: 'The number of filter results returned for each filter type.'
-
     field 'Search Facet Result Sizes',
```

Comment:
the description of this now doesn't make sense since it references the "default config above"

Reply:
In the code "above" refers to the specific filter settings, and it renders that way in the UI too
<img width="950" alt="Screen Shot 2020-09-28 at 8 19 05 AM" src="https://user-images.githubusercontent.com/165111/94431549-6e5e5c80-0163-11eb-843a-c550594ad141.png">

Reply:
Im talking about "search facet result sizes" description, says "the default config above", which is now below it.

Reply:
Oh derp, I've updated.